### PR TITLE
Added note about Anaconda install

### PR DESCRIPTION
**Pull request by jamesacris**
<details>
<summary><strong>Instructions</strong></summary>

Thanks for contributing! :heart:

If this contribution is for instructor training, please email the link to this contribution to
instructor.training@carpentries.org so we can record your progress. You've completed your contribution
step for instructor checkout by submitting this contribution!

Keep in mind that **lesson maintainers are volunteers** and it may take them some time to
respond to your contribution. Although not all contributions can be incorporated into the lesson
materials, we appreciate your time and effort to improve the curriculum. If you have any questions
about the lesson maintenance process or would like to volunteer your time as a contribution
reviewer, please contact The Carpentries Team at team@carpentries.org.

You may delete these instructions from your comment.

\- The Carpentries
</details>

### CODE REVIEW

File: _includes/install_instructions/python.html [hunk 1]
```diff
@@ -14,7 +14,8 @@ <h3>Python</h3>
     well.  Installing all of its research packages individually can be
     a bit difficult, so we recommend
     <a href="https://www.anaconda.com/products/individual">Anaconda</a>,
-    an all-in-one installer.
+    an all-in-one installer. Note that the Anaconda install process can 
+    take a while, so make sure to do it in advance of the workshop.
```
Comment (elichad) [inline _includes/install_instructions/python.html]: I think it's worth quantifying the length of time - I'd also consider five minutes to be a while but not enough of a problem to not leave it to the last minute...
```suggestion
    take 20 minutes or more, so make sure to do it in advance of the workshop.
```

File: _includes/install_instructions/python.html [hunk 2]
```diff
@@ -50,7 +51,7 @@ <h3>Python</h3>
         <ol>
           <li>Open <a href="https://www.anaconda.com/products/individual#download-section">https://www.anaconda.com/products/individual#download-section</a> with your web browser.</li>
           <li>Download the Anaconda for Windows installer with Python 3. (If you are not sure which version to choose, you probably want the 64-bit Graphical Installer <em>Anaconda3-...-Windows-x86_64.exe</em>)</li>
-          <li>Install Python 3 by running the Anaconda Installer, using all of the defaults for installation <em>except</em> make sure to check <strong>Add Anaconda to my PATH environment variable</strong>.</li>
+          <li>Install Python 3 by running the Anaconda Installer, using all of the defaults for installation <em>except</em> make sure to check <strong>Add Anaconda to my PATH environment variable</strong>. Note that the install can take a long time.</li>
```
Comment (elichad) [inline _includes/install_instructions/python.html]:
```suggestion
          <li>Install Python 3 by running the Anaconda Installer, using all of the defaults for installation <em>except</em> make sure to check <strong>Add Anaconda to my PATH environment variable</strong>. Note that the install can take 20 minutes or more.</li>
```

**Comment by jamesacris** [comment]
Cool thanks for the suggestions @elichad, I'll deploy the changes once the tests complete

**Comment by elichad** [comment]
Thanks. Worth making a PR to the upstream template too? https://github.com/carpentries/workshop-template

**Pull request merged**